core/domain/question_domain_test.py

```diff
@@ -104,17 +104,16 @@ def test_create_default_question(self):
         """
 
         question_id = 'col1.random'
-        language_code = 'en'
         question = question_domain.Question.create_default_question(
-            question_id, language_code)
+            question_id).to_dict()
```

Comment:
This seems wrong.  You should be testing the domain object, not the dict. (Especially since the variable is called "question" and not "question_dict".)

Reply:
Yes, I must have missed it. Changed.